includes/discovery/sensors/count/packetfluxpfr.inc.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+#PACKETFLUX-GNSS-MIB::gnss1PPSMissingPulses.0.0
+$oid = '.1.3.6.1.4.1.32050.3.4.1.1.14.0.0';
+$current = (snmp_getnext_multi($device, $oid, '-Oqv') / 1);
```

Comment:
Sorry, I meant doing 1 "snmp multi" request per file, so only one request to go the device. Check other OS for inspiration

Reply:
Understood. Thanks for the clarification!